_posts/2019-06-25-building-a-mono-repo.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Building Multiple Projects in a Monorepo"
+date:   2017-04-06 17:23:53 -0700
+categories: jekyll update
+---
+
+<div align="center">
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">How do people set up Jenkins for their monorepos (... without converting everything to bazel/pants/etc; just many projects in one repo, with different build requirements, interdependent)?</p>&mdash; Dmitriy Ryaboy (@squarecog) <a href="https://twitter.com/squarecog/status/1143678611406241792?ref_src=twsrc%5Etfw">June 26, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</div>
+
+# What is in your repo, and what does it build?
+
+The first question to ask your self is what is the nature of the projects in your monorepo, and what does "build" mean for them. Do you:
```

Comment:
might be better to lead with "Which of these do you need?" and then eliminate the question phrasing in the bullets